### PR TITLE
Webdesigner friendly

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: src/app/js/formats/istexSummary/IstexList.js [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
+        },
     },
-    loadMore: {
-        marginTop: '1.5rem',
-    },
-    skip: {
-        paddingLeft: 0,
-    },
-});
+    'load-more',
```
Comment (Kmaschta) [inline src/app/js/formats/istexSummary/IstexList.js]:
```suggestion
    'istex-list',
```

File: src/app/js/formats/streamgraph/Streamgraph.js [hunk 2]
```diff
@@ -243,7 +247,7 @@ class Streamgraph extends PureComponent {
         const legendView = d3DivContainer
             .append('div')
             .attr('id', 'legend')
-            .attr('class', `${css(styles.legend)}`);
+            .attr('class', `${styles.legend}`);
```
Comment (Kmaschta) [inline src/app/js/formats/streamgraph/Streamgraph.js]: Just remove the template literal

File: src/app/js/formats/streamgraph/Streamgraph.js [hunk 3]
```diff
@@ -252,7 +256,7 @@ class Streamgraph extends PureComponent {
             const element = colorNameList[index];
             const legendItemContainer = legendView
                 .append('div')
-                .attr('class', `${css(styles.legendItem)}`);
+                .attr('class', `${styles.legendItem}`);
```
Comment (Kmaschta) [inline src/app/js/formats/streamgraph/Streamgraph.js]: Just remove the template literal

File: src/app/js/formats/streamgraph/Streamgraph.js [hunk 4]
```diff
@@ -262,12 +266,12 @@ class Streamgraph extends PureComponent {
 
             legendItemContainer
                 .append('text')
-                .attr('class', `${css(styles.legendItemText)}`)
+                .attr('class', `${styles.legendItemText}`)
```
Comment (Kmaschta) [inline src/app/js/formats/streamgraph/Streamgraph.js]: Just remove the template literal

File: src/app/js/lib/stylesToClassName.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { StyleSheet, css } from 'aphrodite/no-important';
+
+export default (styles, componentName) => {
+    const prefix = componentName ? `${componentName}-` : '';
+    const stylesheet = StyleSheet.create(styles);
+    return Object.keys(stylesheet).reduce(
+        (acc, classname) =>
+            Object.defineProperty(acc, classname, {
+                get: () =>
+                    `${css(stylesheet[classname])} ${prefix}${classname}`,
```
Comment (Kmaschta) [inline src/app/js/lib/stylesToClassName.js:10]:
```suggestion
                    `${css(stylesheet[classname])} ${prefix}-${classname}`,
```

File: src/app/js/public/menu/MenuItem.js [hunk 6]
```diff
@@ -3,82 +3,82 @@ import { PropTypes } from 'prop-types';
...
-            color: theme.orange.primary,
+        menuItemIcon: {
+            margin: '0 auto',
         },
     },
-    menuItemIcon: {
-        margin: '0 auto',
-    },
-});
+    'nav-bar',
```
Comment (Kmaschta) [inline src/app/js/public/menu/MenuItem.js]:
```suggestion
    'menu-item',
```

**Review by Kmaschta — CHANGES_REQUESTED** [review]

File: src/app/js/lib/stylesToClassName.js (hunk 5 above)
Comment (ThieryMichel) [reply src/app/js/lib/stylesToClassName.js:10]: prefix is either `''` or `'something-'`, this way it can be optional

**Review by Kmaschta — APPROVED** [review]

**Pull request merged**